### PR TITLE
Fixed issue #9

### PR DESCRIPTION
**Pull request by daquintero**
As described in [this issue #9 ](https://github.com/neo4j-graphql/graphql-cli-load/issues/9), I was trying to run this command in the most minimal Prisma example which I created [here](https://github.com/daquintero/graphql-cli-load/tree/prismaExample/prismaExample): 
```
graphql load --csv reviews.txt --mutation corruptionIn --mapping '{ "url" : "commentary", "description" : "stars"}'
```

With the original graphql-cli-load code, I got this error:
```
✖ 
Syntax Error: Expected Name, found }: {"response":{"errors":[{"message":"Syntax Error: Expected Name, found }","locations":[{"line":3,"column":1}]}],"status":400},"request":{"query":"mutation { \n\n}","variables":{}}}
```
 
After a lot of research into your source code, I recognized the syntax error was because there was no mapped mutations being built inside `buildMutations`, and then saw that the `rMapping` was calling the wrong "key" to select the `values` from the given csv `rows`. Changing this to the original `mapping`, the whole graphql-cli-load worked properly.

### CODE REVIEW

**Comment by jexp** [comment]
Hi,

there is a lot of unrelated changes and debug comments still in there.

Did you also consider the case when there is no mapping passed in? B/c that's what the `rMapping` was for, then `mapping` is not existent.

File: src/load.ts [hunk 1]
```diff
@@ -109,34 +112,48 @@ function getMutation(config, argv) {
...
+
     var fullfilled = true;
     const params = Object.keys(args).map( (key) => { 
       const arg = args[key];
-      const column=(rMapping[key]||key).toString();
+
+
+      //  const column=(rMapping[key]||key).toString(); // original
+      //  console.log("Correcting the key mapping data with const column =  mapping[key].toString() is " + (mapping[key]).toString())
+      const column = mapping[key].toString()
```
Comment (jexp) [inline src/load.ts:143]: what happens if there is no entry for `key` in the mapping? Then the original name from the mutation args has to be used.

**Comment by jexp** [comment]
Please remove all the debug logs, commented out code and whitespace changes, otherwise it's really hard to review the changes.

**Comment by daquintero** [comment]
I had an error with my reference of the json columns, so there was no problem with the source code.

**Pull request closed without merging**